src/common/registry/test/validator-registry/registry-update.spec.ts

```diff
@@ -339,3 +324,90 @@ describe('Empty registry', () => {
     await registryService.update(address, blockHash);
   });
 });
+
+describe('Reorg detection', () => {
+  const provider = new JsonRpcBatchProvider(process.env.PROVIDERS_URLS);
+  let registryService: ValidatorRegistryService;
+  let registryStorageService: RegistryStorageService;
+  let moduleRef: TestingModule;
+  const mockCall = jest.spyOn(provider, 'call').mockImplementation(async () => '');
+  const CHAIN_ID = process.env.CHAIN_ID || 1;
+  const address = REGISTRY_CONTRACT_ADDRESSES[CHAIN_ID];
+  let mikroOrm: MikroORM;
+
+  jest.spyOn(provider, 'detectNetwork').mockImplementation(async () => getNetwork('mainnet'));
```

Comment:
the same

Reply:
I propose to do it as part of another task